visidata/loaders/http.py

```diff
@@ -47,19 +48,24 @@ def _iter_lines(path=path, response=response, max_next=vd.options.http_max_next)
             linkhdr = response.getheader('Link')
             src = None
             if linkhdr:
-                links = urllib.parse.parse_header(linkhdr)
-                src = links.get('next', {}).get('url', None)
+                links = requests.utils.parse_header_links(linkhdr)
+                link_data = {}
+                for link in links:
+                    key = link.get('rel') or link.get('url')
+                    link_data[key] = link
+                src = link_data.get('next', {}).get('url', None)
 
             if not src:
                 break
 
             n += 1
             if n > max_next:
-                vd.warning(f'stopping at max {max_next} pages')
+                vd.warning(f'stopping at max next pages: {max_next} pages')
                 break
 
             vd.status(f'fetching next page from {src}')
-            response = requests.get(src, stream=True, **vd.options.getall('http_req_'))
```

Comment:
Hm, this is weird, were we still trying to use the `requests` library without importing it?

Reply:
Yes, there had previously been two lines with requests.get(), but only the first one had been changed to use urllib.request.urlopen().